Tests/MusicXMLTests/LilyPondTests/Partwise/01_Pitch/Partwise_01_Pitches_A_Pitches.swift

```diff
@@ -7,1270 +7,6 @@
-                <type>quarter</type>
-                <accidental>natural</accidental>
-              </note>
-            </measure>
-            <!--=======================================================-->
-            <measure number="27">
-              <note>
-                <pitch>
-                  <step>C</step>
-                  <alter>2</alter>
-                  <octave>5</octave>
-                </pitch>
-                <duration>1</duration>
-                <voice>1</voice>
-                <type>quarter</type>
-                <accidental>double-sharp</accidental>
-              </note>
-              <note>
-                <pitch>
-                  <step>C</step>
-                  <alter>-2</alter>
-                  <octave>5</octave>
-                </pitch>
-                <duration>1</duration>
-                <voice>1</voice>
-                <type>quarter</type>
-                <accidental>flat-flat</accidental>
-              </note>
-              <note>
-                <pitch>
-                  <step>C</step>
-                  <alter>1</alter>
-                  <octave>5</octave>
-                </pitch>
-                <duration>1</duration>
-                <voice>1</voice>
-                <type>quarter</type>
-                <accidental>sharp</accidental>
-              </note>
-              <note>
-                <pitch>
-                  <step>C</step>
-                  <alter>1</alter>
-                  <octave>5</octave>
-                </pitch>
-                <duration>1</duration>
-                <voice>1</voice>
-                <type>quarter</type>
-                <accidental>sharp</accidental>
-              </note>
-            </measure>
-            <!--=======================================================-->
-            <measure number="28">
-              <note>
-                <pitch>
-                  <step>C</step>
-                  <alter>1</alter>
-                  <octave>5</octave>
-                </pitch>
-                <duration>1</duration>
-                <voice>1</voice>
-                <type>quarter</type>
-                <accidental>sharp</accidental>
-              </note>
-              <note>
-                <pitch>
-                  <step>C</step>
-                  <alter>1</alter>
-                  <octave>5</octave>
-                </pitch>
-                <duration>1</duration>
-                <voice>1</voice>
-                <type>quarter</type>
-                <accidental editorial="yes">sharp</accidental>
-              </note>
-              <barline location="right">
-                <bar-style>light-heavy</bar-style>
-              </barline>
-            </measure>
-          </part>
-          <!--=========================================================-->
-        </score-partwise>
-        """
+        return try! LilyPondTestSuite.string(testName: "01a-Pitches-Pitches", traversal: .partwise)
```

Comment:
Yes, I guess a line like this could go directly into a `XCTestCase` function. Could you use `XCTAssertNoThrow`?

Reply:
Yes, in the end, we don't need a separate file for each of these. We could end up with just a single single for `Partwise` and another for `Timewise`, with a function in each per group of tests.

Reply:
nice!